library/tinkerpop/src/test/java/uk/gov/gchq/gaffer/tinkerpop/GafferPopFederatedIT.java

```diff
@@ -249,13 +249,26 @@ void shouldGetEdgesById() {
 
         assertThat(result)
                 .extracting(item -> item.id().toString())
-                .containsExactly("[p1, s1]", "[p3, s1]");
+                .containsExactly("[p1, created, s1]", "[p3, created, s1]");
 
         assertThat(result)
                 .extracting(item -> item.label())
                 .containsExactly(CREATED_EDGE_GROUP, CREATED_EDGE_GROUP);
     }
 
+    @Test
+    void shouldGetEdgesByIdAndLabel() {
+        // Given
+        GraphTraversalSource g = gafferPopGraph.traversal();
+
+        // When
+        List<Edge> result = g.E("[p1, created, s1]").toList();
```

Comment:
Maybe want to do some testing with some varying white space etc. just to better stress the regex